feature-extraction/ohdsi/feature_extraction/__init__.py

```diff
@@ -1150,14 +1089,15 @@ def create_temporal_covariate_settings(
 
         Returns
         -------
+        ListVector
             An object of type ``covariateSettings``, to be used in other
             functions.
 
         Examples
         --------
         >>> settings = DefaultTemporalCovariateSettings.create_temporal_covariate_settings(
-        ...     use_demographics_gender=True,
-        ...     use_demographics_age=True,
+        ...     use_demographics_gender = True,
+        ...     use_demographics_age = True,
```

Comment:
There should be no spaces arround `=` in keyword arguments. See: https://peps.python.org/pep-0008/#other-recommendations